### DISCUSSION TITLE
[Express.js] This Serverless Function has crashed

### DISCUSSION THREAD

**Discussion by LowScarlet** [vercel/community#2111] [Help] [answered]
### Summary

I am using a source code that is very similar to my old one, and the old one was working without any issues with the message "This Serverless Function has crashed." I have checked the dependencies, including bcryptjs, and confirmed that there are no issues with them. However, in the new source code, I added a dependency "express-validator": "^7.0.1". Could this be causing the issue?

Here is the link that is showing the error: https://sar-website-b19fw60vy-lowscarlet.vercel.app/src/app.js
And here is the link that is working: https://lowscarlet-website-2bra.vercel.app//

### Additional information

![image](https://user-images.githubusercontent.com/64282946/232702961-ef6a554e-7914-411d-8b74-19ceeea78e69.png)
![image](https://user-images.githubusercontent.com/64282946/232703010-c8b56b7f-3611-466c-86c4-1fbfbfcfe03c.png)

### Example

https://sar-website-b19fw60vy-lowscarlet.vercel.app/

**Comment by amyegan** [comment, marked as answer]
It looks like you're already past the point of needing the [Using Express.js with Vercel](https://vercel.com/guides/using-express-with-vercel) guide. Do you see any more details in the [Serverless Function logs](https://vercel.com/docs/concepts/functions/serverless-functions#logging-serverless-functions) or [Runtime Logs](https://vercel.com/docs/concepts/observability/runtime-logs#)? Sometimes I get more info if I wrap the function in a try/catch and `console.log` the error details
